kolibri_explore_plugin/assets/src/app.js

```diff
@@ -12,6 +12,19 @@ Vue.use(IconsPlugin);
 
 Vue.config.productionTip = false;
 
+/** Dynamic load of all eos-components **/
+const requireContext = require.context('eos-components/src/components', false, /.vue$/);
```

Comment:
Why do we need dynamic reload in the Discovery page? We don't have overrides here. If it's just for having the components as globals and not importing them all the time, I think it's better to do a similar thing as Bootstrap-vue does, which is implement a vuejs plugin and then do:

```
Vue.use(EosComponents);
```

The `require.context` is a trick we do only for overrides, and has downsides. For instance, webpack tree shaking won't cut the unused components, making the build bigger than needed.